service/controller/chartconfig/v7/resource/chart/current.go

```diff
@@ -17,7 +18,11 @@ func (r *Resource) GetCurrentState(ctx context.Context, obj interface{}) (interf
 		return nil, microerror.Mask(err)
 	}
 
-	if key.IsCordoned(customObject) {
+	// Delete annotation is set when the chartconfig CR has been migrated to an
+	// app CR and can be safely deleted.
+	_, hasDeleteAnnotation := customObject.Annotations[annotation.DeleteCustomResourceOnly]
```

Comment:
Shouldn't we have a function in the key package as for cordon annotation?

Reply:
Yes good point. I'll add that.